apps/trading-e2e/src/integration/deposits.feature

```diff
@@ -0,0 +1,119 @@
+Feature: Deposits to vega wallet
+
+  Background:
+    Given I navigate to deposits page
+
+  Scenario: Connecting Ethereum wallet
+    Then I can see the eth not connected message "Connect your Ethereum wallet"
+    And the connect button is displayed
+    When I connect my Ethereum wallet
+    Then I can see the deposit form
+
+  @todo
+  Scenario: Cannot deposit if approved amount is 0 (approval amount is 0)
+    And I connect my Ethereum wallet
+    When I set "0" tokens to be approved
+    And I approve the asset tokens
+    And I can see the deposit form is displayed
+    And I select "" asset from the dropdown list
+    When I enter the following details
+      | Field         | Value    |
+      | To (Vega key) | xxxxxxxx |
+      | Amount        | 50       |
+    And I click to the deposit the funds
+    And I approve the ethereum transaction
+  # The following step is valid and are commented out as currently it cannot be automated
+  # Then I can see the deposit is unsuccessful
+
+  @todo
+  Scenario: Cannot deposit if approved amount is lower than deposit amount
+    When I set "2" tokens to be approved
+    And I approve the asset tokens
+    And I can see the deposit form is displayed
+    And I select "" asset from the dropdown list
+    When I enter the following details
+      | Field         | Value    |
+      | To (Vega key) | xxxxxxxx |
+      | Amount        | 50       |
+    And I click to the deposit the funds
+    And I approve the ethereum transaction
+  # The following step is valid and are commented out as currently it cannot be automated
+  # Then I can see the deposit is unsuccessful
+
+  @todo
+  Scenario: Can succesfully deposit (approved amount is greater than deposit)
+    When I set "200000000" tokens to be approved
+    And I approve the asset tokens
+    And I can see the deposit form is displayed
+    And I select "" asset from the dropdown list
+    When I enter the following details
+      | Field         | Value    |
+      | To (Vega key) | xxxxxxxx |
+      | Amount        | 50       |
+    And I click to the deposit the funds
+    And I approve the ethereum transaction
+  # The following steps are valid and are commented out as currently they cannot be automated
+  # Then I can see the deposit is Successfull
+  # And Balance is updated to reflect deposit amount
+
+  Scenario: Validation errors
+    Given I connect my Ethereum wallet
+    When I submit a deposit with empty fields
+    Then I can see validation errors present
+    And I enter an invalid public key
+    Then Invalid Vega key is shown
+    And I enter an amount less than the minimum viable amount
+    Then Amount too small message shown
+    And I enter a valid amount
+    Then Not approved message shown
```

Comment:
Chore(non-blocking): For visibility can we move this up below the other test that is running?